skimage/filters/ridges.py

```diff
@@ -208,7 +213,8 @@ def meijering(image, sigmas=range(1, 10, 2), alpha=None,
     for i, sigma in enumerate(sigmas):
 
         # Calculate (sorted) eigenvalues
-        eigenvalues = compute_hessian_eigenvalues(image, sigma, sorting='abs')
+        eigenvalues = compute_hessian_eigenvalues(image, sigma, sorting='abs',
+                                                  mode=mode, cval=cval)
```

Comment:
In this case, there will be no warning about the change of behaviour to meijering. Do we want to add this to the release notes under "API changes"?

Reply:
Here I considered that it was a bug fix since `meijering` was suffering from artifacts on the image borders, that why I did not raised any warning.
What about adding the API changes for all the ridge filters in the release note?